tests/__init__.py

```diff
@@ -0,0 +1,37 @@
+from typing import IO, Any, Mapping, Optional, Sequence, Union
+
+import click.core
+import click.testing
+
+
+def reset_common() -> None:
```

Comment:
could we add info into this file, or something, so it is cleared why this was added please?

Reply:
Covered.